interface/patient_file/ccr_pending_approval.php

```diff
@@ -47,65 +53,68 @@
-            if (sqlNumRows($query) > 0) {
-                while ($res = sqlFetchArray($query)) {
-                    $dup_query = sqlStatement("SELECT * FROM audit_master am JOIN audit_details ad ON ad.audit_master_id = am.id AND ad.table_name = 'patient_data' AND ad.field_name = 'lname' JOIN audit_details ad1 ON ad1.audit_master_id = am.id AND ad1.table_name = 'patient_data' AND ad1.field_name = 'fname' JOIN audit_details ad2 ON ad2.audit_master_id = am.id AND ad2.table_name = 'patient_data' AND ad2.field_name = 'DOB' JOIN patient_data pd ON pd.lname = ad.field_value AND pd.fname = ad1.field_value AND pd.DOB = DATE(ad2.field_value) WHERE am.id = ?", array($res['amid']));
-                    ?>
-            <tr>
-                <td class="font-weight-bold">
-                    <?php echo text($res['pat_name']); ?>
-                </td>
-                    <?php
-                    if (sqlNumRows($dup_query) > 0) {
-                        $dup_res = sqlFetchArray($dup_query);
-                        ?>
-                <td align="center" class="font-weight-bold">
-                        <?php echo xlt('Yes'); ?>
-                </td>
-                <td align="center" >
-                    <a href="ccr_review_approve.php?revandapprove=1&amid=<?php echo attr_url($res['amid']); ?>&pid=<?php echo attr_url($dup_res['pid']); ?>&csrf_token_form=<?php echo attr_url(CsrfUtils::collectCsrfToken()); ?>" class="btn btn-primary" onclick="top.restoreSession()" ><?php echo xlt('Review & Approve'); ?></a>
-                </td>
-                        <?php
-                    } else {
-                        ?>
-                <td align="center" class="font-weight-bold">
-                        <?php echo xlt('No'); ?>
-                </td>
-                <td align="center">
-                    <a href="ccr_pending_approval.php?approve=1&am_id=<?php echo attr_url($res['amid']); ?>&csrf_token_form=<?php echo attr_url(CsrfUtils::collectCsrfToken()); ?>" class="btn btn-primary" onclick="top.restoreSession()" ><?php echo xlt('Approve'); ?></a>
-                </td>
-                        <?php } ?>
-            </tr>
-                    <?php
-                }
-            } else {
-                ?>
-                <tr>
-                    <td colspan="3">
-                        <?php echo xlt('Nothing Pending for Approval') . "."; ?>
-                    </td>
-                </tr>
-                <?php
-            }
-            ?>
-        </table>
+<body>
+    <div class="container mt-3">
+        <div class="row">
+            <div class="col-12">
```

Comment:
Don't need row or col-12 here cause we are already at 100% width

Reply:
Hi @tywrenn 

If we remove the `.col-12` class, not only h2 can't align center, but also caused the form overflow.
![image](https://user-images.githubusercontent.com/22230889/89750819-20808d00-db00-11ea-8faa-773ed19c864c.png)

Reply:
That is really weird. It shouldn't be doing that when you remove the col-12. There are multiple ones in this file so that might be causing issues. If that doesn't work i'll play around with it.

Reply:
I check again, but it still looks like same, maybe needs your check,